media/main.js

```diff
@@ -79,36 +90,44 @@ function () {
 				const requirementName = json.requirementName;
 				const isAvailable = json.result;
 
+				let requirementMessage = 'Not currently available';
 				let element = document.getElementById(requirementName);
```

Comment:
element can be made a const and given a type

Reply:
const maybe, but see the notes about JS vs. TS in this case, so no type

Reply:
my fault